### PR TITLE
[Makefile] Point to xcode12.1 xip file.

### PR DESCRIPTION
**Pull request by mandel-macaque**
RE provided the 12.1 version.

### CODE REVIEW

**Comment by mandel-macaque** [comment]
@monojenkins backport d16-8

**Review by tj-devel709 — APPROVED** [review]

File: Make.config [hunk 1]
```diff
@@ -112,7 +112,7 @@ WATCHOS_NUGET_VERSION_FULL=$(WATCHOS_NUGET_VERSION_NO_METADATA)+$(NUGET_BUILD_ME
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
 XCODE_VERSION=12.1
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_12.1_GM_seed.xip
+XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_12.1.xip
 XCODE_DEVELOPER_ROOT=/Applications/Xcode_12.1.0-GM.app/Contents/Developer
```
Comment (spouliot) [inline Make.config:116]: ^ don't you want/need this updated for the device bots ?

Comment (mandel-macaque) [reply Make.config:116]: @spouliot the device bots use provisioning, the provisionator file gets updated by just changing this Make.config. I use a Make file to generate the csx file based on the configuration in the Make.config :)

**Review by spouliot — APPROVED** [review]

**Review by rolfbjarne — APPROVED** [review]

**Comment by premun** [comment]
@mandel-macaque not sure if this helps in any way but @greenEkatherine will be re-uploading the xip with `Xcode_12.1.xip` name soon.

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**